src/index.ts

```diff
@@ -58,25 +52,11 @@ export class IDX {
     return this._ceramic
   }
 
-  get did(): DID {
-    if (this._ceramic.did == null) {
-      throw new Error('Ceramic instance is not authenticated')
-    }
-    return this._ceramic.did
-  }
-
   get id(): string {
```

Comment:
Does it still make sense to keep this method?

Reply:
I think it's still useful, same as `idx.authenticated` for convenience but they're not necessary if you think it's better to remove?